cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -683,6 +683,215 @@ const char* gdv_fn_upper_utf8(int64_t context, const char* data, int32_t data_le
   return out;
 }
 
+GDV_FORCE_INLINE
+uint64_t unsigned_long_div(gdv_int64 x, gdv_int32 m) {
```

Comment:
Use `int64_t` instead `gdv_int64`to keep the code easier to read

Reply:
Done